draft/src/main.rs

```diff
@@ -0,0 +1,84 @@
+mod random_drafter;
+mod api;
+use crate::api::*;
+extern crate clap;
+use clap::{Arg, App};
+use std::fs::File;
+use std::io::{prelude::*, BufReader};
+use rand::seq::SliceRandom;
+use rand::Rng;
+
+fn main() {
+    let matches = App::new("Drafting")
+        .arg(Arg::with_name("list")
+                    .short("l")
+                    .long("list")
+                    .help("Text file containing full cube list (one card name per line)")
+                    .takes_value(true)
+                    .required(true))
+        .get_matches();
+
+    let list_filename = matches.value_of("list").unwrap().to_string();
+
+    println!("list filename: {}", list_filename);
+
+    let draft_info = create_draft_info(&list_filename);
+    let mut draft_packs = create_draft_packs(&draft_info);
+
+    println!("draft packs: {:?}", draft_packs);
+
+    // TODO: Implement full draft of random drafters
+    let drafter = random_drafter::RandomDrafter {};
+
+    let pack = &draft_packs[0][0];
+    let cards_owned : Vec<api::Card> = vec![];
+
+    println!("Random drafter's pick: {}", drafter.pick(&pack, &cards_owned, &draft_info));
+}
+
+fn create_draft_info(cube_list_filename: &String) -> DraftInfo {
+    // from https://stackoverflow.com/a/35820003
+    let list_file = File::open(cube_list_filename).expect("file not found");
+    let buf = BufReader::new(list_file);
+    let cube_list: Vec<Card> = buf.lines()
+        .map(|l| Card::new(&l.expect("Could not parse line")))
+        .collect();
+
+    DraftInfo {
+        card_list: cube_list,
+        num_drafters: 6,
+        num_packs: 3,
+        cards_per_pack: 15
+    }
+}
+
+// Returns shuffled draft packs in the following format:
+// Outermost vector: which pack (e.g. drafts normally have 3 packs per player)
```

Comment:
Comment is ambiguous; pack refers both to the object and to the round or phase of draft.

Reply:
This is a tough comment to solve nicely; I'm not certain I can get it better than it already is.

Reply:
Took your suggestion on "draft phases" instead of "pack".